### DISCUSSION TITLE
How can I create my own price that will be visible to the seller?

### DISCUSSION THREAD

**Discussion by Vearodev** [azerothcore/azerothcore-wotlk#8995] [General]
How can I create my own price that will be visible to the seller?

I added a price to the dbc with this item. 
https://ru.wowhead.com/item=37711/currency-token-test-token-1

The price works, but it is not visible from the seller.

the object itself in the game is visible.

**Comment by Eronox** [comment]
DBC files are client side, so you will either have to make a patch for it and force your players to use it, or you can change existing items currency to be a unused-item count
